spec/src/main/asciidoc/release-notes.adoc

```diff
@@ -33,6 +33,7 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 ==== API/SPI Changes
 * Consume the OpenTelemetry Java release link:https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.39.0[v1.39.0]. The full comparison with the link:https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.29.0[v1.29.0] supported by MicroProfile Telemetry 1.1 can be found link:https://github.com/open-telemetry/opentelemetry-java/compare/v1.29.0...v1.39.0[here].
 * Adopt OpenTelemetry Metrics API (https://github.com/eclipse/microprofile-telemetry/issues/141[141], https://github.com/eclipse/microprofile-telemetry/issues/149[149])
+* Expose OTel APIs for a better user experience (https://github.com/eclipse/microprofile-telemetry/issues/210[210])
```

Comment:
This makes it sound like the spec requires APIs to be available to applications which previously weren't.

```suggestion
* Add an API maven artifact which depends on the relevant Open Telemetry API artifacts (https://github.com/eclipse/microprofile-telemetry/issues/210[210])
```